evalml/model_understanding/visualizations.py

```diff
@@ -419,6 +446,53 @@ def graph_prediction_vs_actual_over_time(pipeline, X, y, X_train, y_train, dates
         dates,
     )
 
+    if is_multiseries(pipeline.problem_type):
+        id_list = (
+            [single_series] if single_series is not None else data["series_id"].unique()
+        )
+        fig = subplots.make_subplots(
+            rows=len(id_list),
+            cols=1,
+            subplot_titles=[f"Series: {id}" for id in id_list],
+        )
+        for curr_count, id in enumerate(id_list):
+            curr_df = data[data["series_id"] == id]
+            fig.append_trace(
+                _go.Scatter(
+                    x=curr_df["dates"],
+                    y=curr_df["target"],
+                    mode="lines+markers",
+                    name=f"Series {id}: Target",
+                ),
+                row=curr_count + 1,
+                col=1,
+            )
+            fig.append_trace(
+                _go.Scatter(
+                    x=curr_df["dates"],
+                    y=curr_df["prediction"],
+                    mode="lines+markers",
+                    name=f"Series {id}: Prediction",
+                ),
+                row=curr_count + 1,
+                col=1,
+            )
+            fig.update_xaxes(title_text="Time")
+            fig.update_yaxes(title_text=y.name)
+        if single_series is not None:
+            fig.update_layout(
+                height=600,
```

Comment:
is there any autosizing we can take advantage of? Also not sure if we need the single_series case: can it just match what we already had before for time series?

Reply:
> Is there any autosizing we can take advantage of?

- For python plotly, it doesn't look like there's any autosizing besides doing what I did in line 490 (though correct me if I'm wrong). If I left the parameters blank for the multiseries graph (i.e. don't define x and y) and let it use the default values the graph looks really squished.
<img width="1211" alt="Screen Shot 2023-08-21 at 11 33 29 AM" src="https://github.com/alteryx/evalml/assets/56745347/aa17756f-6598-46a7-993a-57410b03559b">

- For single series though I can probably leave the parameters blank since it is only one graph.

> Also not sure if we need the single_series case: can it just match what we already had before for time series?

- The issue is that for single_series case I want the data that corresponds to that single series. 
    - In the before case for time series, y is taken from `data["target"]` whereas for single series I want to take it from `curr_df["target"]` where `curr_df = data[data["series_id"] == single_series]`
    - If I wanted to do that with the before code, I'd probably have to add more `if` statements and such so it made more sense to me to have it included with the multiseries case.
    - Also I'd like for the single series plot to have a different title than the time series plot.